### PR TITLE
Essence of brand identity

### PR DESCRIPTION
**Pull request by pimatco**
**Description**

### CODE REVIEW

File: src/pages/product-design/brand-identity/index.md [hunk 1]
```diff
@@ -3,13 +3,11 @@ title: Brand Identity
 ---
 ## Brand Identity
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/product-design/brand-identity/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+Make sure you create a brand not only with the logo. Brand identity is about all usages and applications a brand may have passing its essence with ease.
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+It's a manual showing where the identity may be inserted, based on design patterns.
```
Comment (jp-sauve) [inline src/pages/product-design/brand-identity/index.md]: What is the "it" you say is a manual?

File: src/pages/product-design/brand-identity/index.md [hunk 2]
```diff
@@ -3,13 +3,11 @@ title: Brand Identity
 ---
 ## Brand Identity
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/product-design/brand-identity/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+Make sure you create a brand not only with the logo. Brand identity is about all usages and applications a brand may have passing its essence with ease.
```
Comment (jp-sauve) [inline src/pages/product-design/brand-identity/index.md]: When you say "not only", you need to follow with some kind of "also".

File: src/pages/product-design/brand-identity/index.md (hunk 1 above)
Comment (pimatco) [reply src/pages/product-design/brand-identity/index.md]: I mean to have. Brand identity has a manual.

**Comment by joshalling** [comment]
@pimatco I'm closing this because the requested changes were never made. If you still want to contribute this information please submit a new PR.

**Pull request closed without merging**